sprokit/src/bindings/python/modules/registration.cxx

```diff
@@ -70,6 +70,16 @@ static bool is_suppressed();
 static void _load_python_library_symbols();
 
 
+class scoped_save_thread
```

Comment:
Also minor, but you might want to move this into the python_util folder

Reply:
This class is just to work around the (possible) bug in pybind11, and I made it a class instead of just calling `PyEval_InitThreads()` outright so that it would still be called even if `gil_scoped_acquire` throws an exception. It will not, and should not, be used anywhere but here.

Reply:
An alternative would be:

```c++
try
{
  gil_scoped_acquire acquire;
}
catch( ... )
{
  PyEval_SaveThread();
  throw;
}
PyEval_SaveThread();
```

Reply:
Maybe a comment saying this then